### ISSUE TITLE
Dark mode not getting implemented in Tiles.js

### ISSUE THREAD

**Issue by aswin2108** [tejas-k3/tejas-kothari-portfolio#20]
**Describe the bug**
Currently the `isDarkMode` is having `undefined` value in TileStyle.js file, because of which the dark mode changes does not appear in action. The background, title color and toggle icon are changing and thats because they are present in the App.js file. The tile icon is changing as its in Tile.js. 

**To Reproduce**
You can reproduce this with an existing style statement
1. Go to TileStyles.js
2. Change line 19th to the following
![image](https://github.com/tejas-k3/tejas-kothari-portfolio/assets/72661784/e4ef7a1e-51c0-487a-8c5f-3975e673d554)

**Expected behavior**
In light mode its working fine, because the value is undefined so it executes the false statement so a black shadow appears. Here in the above image I gave a white shadow in dark mode, but no shadow appeared.

| Expected | Output |
| ------------- | ------------- |
| ![image](https://github.com/tejas-k3/tejas-kothari-portfolio/assets/72661784/a473b7f1-3845-482b-bbdd-938dd650ddb4) | ![image](https://github.com/tejas-k3/tejas-kothari-portfolio/assets/72661784/c370ce62-bed9-4ac9-a79d-0e90dcb39ffa) |

**Additional context**
Fixing this will allow us to add different effects to the tiles.

**Comment by siddhi-gh** [comment]
@aswin2108 I purposely kept 'none' value for `boxShadow` in dark mode. 
I don't think shadows appear in a black background.. you can add border css if you want to highlight the tiles in dark mode.

**Comment by siddhi-gh** [comment]
I see that I passed an empty string for `isDarkMode` - you can update the existing code like this:
`boxShadow: isDarkMode ? "none" : "5px 5px 5px #111111",`

**Comment by aswin2108** [comment]
Yeah I will update it! Actually the thing was that I was trying out something in my version of the app, and noticed that there was some issue with it, so to demonstrate it I used the `boxShadow` as it was readily available!

**Comment by tejas-k3** [comment]
@siddhi-gh @aswin2108 between no shadows & white shadow for dark background, let's lock in preferred visuals.

I am looking forward to white shadow as it'll keep the theme element intact
